editor/editor_help.cpp

```diff
@@ -1054,25 +1073,27 @@ void EditorHelp::_update_doc() {
 			_push_code_font();
 
 			if (!cd.properties[i].default_value.is_empty()) {
-				class_desc->push_color(theme_cache.symbol_color);
 				if (cd.properties[i].overridden) {
+					class_desc->push_color(get_theme_color(SNAME("warning_color"), EditorStringName(Editor)));
```

Comment:
I'd suggest adding this as a separate theme cache entry defaulting to the same color

Reply:
Overridden properties rarely come into play, so I felt like doing it was a bit overkill. But it can be done.

Reply:
I'm just not sure the specific color here, using `warning_color` is fully reliably readable on different themes

Reply:
I'm not sure I understand what you are suggestions. There have been some issues with the light theme and the warning color, but they're unrelated to the PR itself.

Reply:
I mean that like the `value_color` this should have a dedicated configurable color, specific for this

Also think it doesn't work quite well on the light theme:
![image](https://github.com/godotengine/godot/assets/96648715/cd397713-3933-46d9-9d84-7ad12cf83a69)

Reply:
Yyyyeah. For light theme, this would inherently be fixed by https://github.com/godotengine/godot/pull/88058

Meanwhile, I updated the PR to use a customizable `override_color`.